src/app/_components/myfeed/PostCardFooter.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, Chip, IconButton } from "@mui/material";
+import React from "react";
+import FavoriteBorderIcon from "@mui/icons-material/FavoriteBorder";
+import MessageIcon from "@mui/icons-material/Message";
+import SendIcon from "@mui/icons-material/Send";
+import TollIcon from "@mui/icons-material/Toll";
+import BookmarkBorderIcon from "@mui/icons-material/BookmarkBorder";
+
+import { type PostCardFooterProps } from "~/app/myfeed/types/types";
+
+const PostCardFooter: React.FC<PostCardFooterProps> = ({
+  likes,
+  comments,
+  shares,
+  bids,
+}) => {
+  return (
+    <Box
+      sx={{
+        width: "100%",
+        display: "flex",
+        justifyContent: "space-between",
+        alignItems: "center",
+      }}
+    >
+      <Box
+        sx={{
+          display: "flex",
+          justifyContent: "start",
+          alignItems: "center",
+          gap: "10px",
+        }}
+      >
+        <Chip
```

Comment:
use the custom chip buttons created here. Add the callbacks in the custom buttons created